src/test/java/com/gooddata/collections/UriPageTest.java

```diff
@@ -38,6 +38,8 @@ public void test() throws Exception {
         final UriComponentsBuilder builder = fromUriString("/this/is/{template}").query("other=false");
 
         uri.updateWithPageParams(builder);
+        uri.updateWithPageParams(builder);
```

Comment:
Well `testUpdateWithPageParamsIdempotency` tests the `PageRequest` object while this tests `UriPage`... As far as I can see, these objects are two separate implementations of `Page`, so they deserve two separate tests.